jekyll/_cci2/status-badges.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: classic-docs
+title: Embedding Build Status Badges
+---
+
+## Overview
+
+CircleCI provides build status badges for both projects and branches. Users tend to embed status badges in GitHub READMEs, although they can be placed virtually anywhere.
```

Comment:
remove 'both' for clarity. 
change Users to 'It is common to embed...'

Reply:
Compromise? "Status badges are commonly embedded..."